src/systemtest/resources/JenkinsShellScripts/TCSTCFScript.txt

```diff
@@ -0,0 +1,97 @@
+import groovy.json.JsonSlurper
+def baseUrl = "http://10.0.2.15:8096";
+
+////////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////TCS1/////////////////////////////////////////////
+////////////////////////////////////////////////////////////////////////////////////////
+
+def uriPathTCS1 = baseUrl + '/generateAndPublish?mp=eiffelsemantics&amp;msgType=EiffelTestCaseStartedEvent';
+def jsonTCS1 = """
+  {
+    "msgParams": {
+      "meta": {
+        "type": "EiffelTestCaseStartedEvent",
+        "version": "1.1.0",
+        "tags": [],
+        "source": {
+          "domainId": "",
+          "host": "",
+          "name": "",
+          "uri": ""
+        },
+        "security": {
+        }
+      }
+    },
+    "eventParams": {
+      "data": {
+        "executor": ""
+      },
+      "links": [
+        {
+          "type": "TEST_CASE_EXECUTION",
+          "target": \\""""+ env.EVENT_ID + """\\"
+        }
+      ]
+    }
+  }
+""";
+
+generateEiffelEventAndPublish(uriPathTCS1, jsonTCS1);
```

Comment:
The method returns an event ID but do we not need it in this script?

Reply:
It's only required in some of the scripts, i just reuse the same method.